README.md

```diff
@@ -35,9 +35,11 @@ Then with a module bundler like [Rollup](https://github.com/rollup/rollup) or [W
 import picostyle from "picostyle"
 ```
 
-Otherwise, download the [latest release](https://github.com/picostyle/picostyle/releases/latest) or load directly from [unpkg](https://unpkg.com/picostyle).
+Otherwise, download the [latest release](https://github.com/picostyle/picostyle/releases/latest) or load directly from [unpkg](https://unpkg.com/picostyle) or [jsDelivr](https://cdn.jsdelivr.net/npm/picostyle@0.1.5/dist/picostyle.js).
```

Comment:
Hi @LukasDrgon! Can you update the link to:

https://cdn.jsdelivr.net/npm/picostyle@latest/dist/picostyle.js

🙏

Reply:
Done :+1:

Reply:
@LukasDrgon I think you missed one of the links.

Reply:
@LukasDrgon Yep, please fix the link here 🙏